components/automate-chef-io/content/docs/install.md

```diff
@@ -156,8 +156,14 @@ Add the following to your config.toml:
 # [global.v1.external.elasticsearch.ssl]
 #  Specify either a root_cert or a root_cert_file
 #  root_cert = """$(cat </path/to/cert_file.crt>)"""
-#  root_cert_file = "</path/to/cert/file>"
 #  server_name = "<elasticsearch server name>"
+
+# Uncomment and fill out if using external elasticsearch that uses hostname-based routing/load balancing
+# [esgateway.v1.sys.ngx.http]
+#  proxy_set_header_host = "yourawesomeexternaleshostname:1234"
```

Comment:
Is the "1234" supposed to be an example port number? Or the exact port to be used?

Reply:
the `1234` is an example port number, it can really be anything, with some Cloud Providers, the port number is 443, or 80, or 92xx or something similar